### DISCUSSION TITLE
Results to be announced early next week

### DISCUSSION THREAD

**Discussion by OmPrakash95** [DiveHQ/backend-internship-task#48] [Announcements]
Hello Hackers! Thank you for your submissions and feedback on the Octernship task. We hope you enjoyed working on it and learned a lot.

Regarding the shortlisting, the response has been overwhelming and our team has been evaluating the submissions.

We look forward to announcing the shortlists early next week. All the best!

**Comment by thunder-007** [comment]
Enjoyed

**Comment by subhendudash02** [comment]
Hey @OmPrakash95 @ashwinexe is the result announced already?

**Comment by TiwariAbhishek23** [comment]
@OmPrakash95 @ashwinexe, hello, I am also waiting for the result and please also provide feedback for my repo [link](https://github.com/DiveHQ-Octernships/dive-backend-engineering-octernship-AbhishekTiwari23), I think admin can access it.

**Reply by TiwariAbhishek23** [reply]
And if anyone else wants to provide feedback, please have a look at [CaloriCo](https://github.com/AbhishekTiwari23/CaloriCo). It is an extended version of the same project

**Comment by shuklaritvik06** [comment]
Yes, I am also waiting for the results

**Comment by swapnalshahil** [comment]
Hmm, waiting for the good things. 😅
Maintainers please provide feedback!

**Comment by asifrahaman13** [comment]
> Hello Hackers! Thank you for your submissions and feedback on the Octernship task. We hope you enjoyed working on it and learned a lot.
> 
> Regarding the shortlisting, the response has been overwhelming and our team has been evaluating the submissions.
> 
> We look forward to announcing the shortlists early next week. All the best!

@OmPrakash95 @ashwinexe 

Are the shortlisting of candidates done?

**Reply by swapnalshahil** [reply]
Did everyone get feedback?

**Reply by asifrahaman13** [reply]
At least I didn't get any feedback.

**Reply by goofynugtz** [reply]
I didn't get a feedback either.

**Comment by AmitChigare** [comment]
Any update regarding the results? @ashwinexe @OmPrakash95 @prayashm @shuklaritvik06

**Reply by shuklaritvik06** [reply]
I didn't got any update :)

**Reply by swapnalshahil** [reply]
I think this organization is new and yet not clear with its goals. We should look somewhere else.

**Reply by OmPrakash95** [reply]
Hi @swapnalshahil - Sorry for the delay. This is @divehq's 3rd Octernship in case if you're wondering we are new to this :D but this time around we increased our submission window to 500 and we are overwhelmed with the response. 

Our team has been spending ample time with each of the submissions to fair evaluation. The team will share a prompt feedback in 2-3 days.

**Reply by shuklaritvik06** [reply]
We are excited for the feedback 🤩

**Reply by priyapahwa** [reply]
> The team will share a prompt feedback in 2-3 days.

Thanks for the update, looking forward to it :)

**Reply by ashwinexe** [reply]
https://github.com/DiveHQ/ml-engineering-intern-task/discussions/15

**Comment by suyogkokaje** [comment]
Could you please provide an update on when the results will be announced? @OmPrakash95 @ashwinexe.

**Reply by ashwinexe** [reply]
https://github.com/DiveHQ/ml-engineering-intern-task/discussions/15
